File: Go/problem010.go
package projecteuler

// #10 Summation of primes - Project Euler
// http://projecteuler.net/index.php?section=problems&id=10
import (
	"log"
	"projecteuler/my"
)

func P010() {
	var ans int
	for _, v := range my.Primes(2000000) {
		ans += v
	}

	log.Println(ans)
}

File: TypeScript/problem010.ts
/*
 * #10 Summation of primes - Project Euler
 * http://projecteuler.net/index.php?section=problems&id=10
 */
import { sieve } from "./prime.ts";
{
    const ans = sieve(2_000_000).reduce((acc, x) => acc + x, 0);
    console.log(ans);
}
